stripe/test/resources/test_ephemeral_keys.py

```diff
@@ -0,0 +1,57 @@
+import stripe
+from stripe.test.helper import StripeResourceTest
+
+
+class EphemeralKeyTest(StripeResourceTest):
+
+    def test_create(self):
+        stripe.EphemeralKey.create(customer='cus_123',
+                                   api_version='2017-05-25')
+
+        self.requestor_class_mock.assert_called_with(
+            None,
+            api_version='2017-05-25',
+            account=None
+        )
```

Comment:
So, this didn't work because `Stripe-Version` isn't included in the call to `request` since it's configured on the `APIRequestor` itself.  In fact, this test would be failing if it worked the other way, since it's doing a deep comparison of the objects (rather than just checking the keys specified).

I figured any other solution would need to change more code, so I erred on the side of the smallest change.